File: kotlin/fizzbuzz1.kt

fun main(){
    for(i in 0..100){
        var fzbz = false
        if(i % 3 == 0){
            fzbz = true
            print("Fizz")
        }
        if(i % 5 == 0){
            fzbz = true
            print("Buzz")
        }
        if(!fzbz){
            print(i)
        }
        println()
    }
}

File: lua/fizzbuzz1.lua

for i=0,100 do
    fzbz = false
    if i % 3 == 0 then
        fzbz = true
        io.write("Fizz")
    end
    if i % 5 == 0 then
        fzbz = true
        io.write("Buzz")
    end
    if not fzbz then
        io.write(i)
    end
    print()
end
